### PR TITLE
Add service docs for PIRI service + Services section

### LINKED ISSUE #12: Add documentation page for PIRI service / add Services list to docs
It'd be good to have a page in the docs for the PIRI service that we could point folks to and update as the service evolves. For background, here are some links to more about the service.

- Redmine ticket: https://redmine.dataone.org/issues/8865
- Draft design doc: https://hpad.dataone.org/8h3o_7VPTIibo5xL9bz24w

I wasn't sure where to put it so I took a look at the top level headers:

- API Reference
- Design Documents
- Notes and Miscellanea
- Implementation Notes
- Indices, Tables, and Change Log

A service description doesn't seem to really fit into any of these so I propose a new top-level header to go after API Reference of **Services**. In this header, we could describe other services such as the quality and metrics services, or Bookkeeper.

I'll submit a PR that adds this top level header and my service description doc for the PIRI space but I'd like to get others' ideas about whether they like this idea.

(cc @datadavev)

### PR DESCRIPTION
**Pull request by amoeba**
Closes https://github.com/DataONEorg/api-documentation/issues/12.

### CODE REVIEW

File: source/services/piri_service.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+The DataONE PIRI Service defines an IRI space with the following pattern:
+
+`https://dataone.org/{type}/{id}`
+
+Where `{type}` is one of:
+
+- `datasets`
+- `people` (not implemented)
+
+And `{id}` is an appropriate identifier for resource such as a PID or SID for the 'datasets' type.
```
Comment (mbjones) [inline source/services/piri_service.txt]: I would add a statement here about expectations around URI escaping of identifiers, and reference the relevant RFCs. There is a fairly extensive treatment of it in the "Serializing" section of https://dataone-architecture-documentation.readthedocs.io/en/latest/design/PIDs.html

**Review by mbjones — APPROVED** [review]
Looks good to me @amoeba I suggested a minor addition on escaping.

File: source/services/piri_service.txt (hunk 1 above)
Comment (amoeba) [reply source/services/piri_service.txt]: Great point, I'll get that in.

**Comment by amoeba** [comment]
Changes WRT escaping made. @datadavev what do you think about adding a Services section like this as mentioned in https://github.com/DataONEorg/api-documentation/issues/12?

File: source/services/piri_service.txt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- `datasets`
+- `people` (not implemented)
+
+And `{id}` is an appropriate identifier for resource such as a PID or SID (Series Identifier) for the 'datasets' type.
+Note that all identifiers should be escaped according to RFC3986_.
+See `Identifiers in DataONE`_ for more information.
+
+.. _RFC3986: https://tools.ietf.org/html/rfc3986
+.. _Identifiers in DataONE: https://dataone-architecture-documentation.readthedocs.io/en/latest/design/PIDs.html
```
Comment (mbjones) [inline source/services/piri_service.txt]: Thanks for adding this. The absolute URI you used might break, so could you switch to using an inline link to the documentation, which will set up a relative link? The syntax is ":doc:`design/PIDs`", and means that links will work even when the docs are built and deployed on different sites. There might be a few of these in your text.

Comment (amoeba) [reply source/services/piri_service.txt]: Good catch.

**Comment by datadavev** [comment]
This seems fine to me.

**Comment by amoeba** [comment]
Great, thanks @datadavev. And thanks @mbjones for the edits.

**Comment by amoeba** [comment]
@mbjones or @datadavev I don't seem to have permissions to merge. Can one of you merge or give me access?

**Comment by amoeba** [comment]
Thanks @datadavev

**Pull request merged**